### ISSUE TITLE
Add accounts to lists without having them in your main timeline

### ISSUE THREAD

**Issue by allo-** [mastodon/mastodon#7876]
I can only add accounts to lists, when I follow them and they are removed from lists when I unfollow them.
I would like to be able to add people to lists without following them.

1. The main implication is, that you have a very full main timeline when you follow accounts which post a lot, while you only want to have them in a list you read when you have the time to read it.
2. Another implication is privacy. I.e. on Twitter you can add people to a private list and they do not get a notification about it. I think on mastodon people get notifications you're following them even when your account is private and/or has the follower/following lists set as private.

I know that the second point is not to make secure anyway (as your instance needs to tell the instance of the other user, that it needs the toots), but I wanted to list it for completeness. On the other hand, you need to trust both instances anyway when you for example toot for followers only.

The first point is why I opened the issue. I want for example a list of webcomics, which I can read when I have more the time to do so, while my main timeline has more actual people in it I want to read all the time.

**Comment by nightpool** [comment]
I don't think this is ever something were going to implement, per the rationale on the original issue for lists: https://github.com/tootsuite/mastodon/issues/134

> I don't like the idea of not knowing if someone is tracking me. Sure, they can "track" me in the Local and Federated Timelines, but that's a morass to filter through. You would also likely need to notify an account they have been added to a list, and provide the account a way to prevent that from happening, whether per list or altogether. You'd also need to put in rules for not allowing a blocked account from adding your account to a list, prevent private accounts from being added to any lists or only being added to lists by people they already allowed to follow them, etc. I think a lot of edge cases can be fixed by only allowing followed accounts to be added to lists.

ideally people who use lists would also be able to exlude accounts from their Home timeline, but in the meantime you can create a list called "Home", add people to it, and then unpin the actual home timeline

**Comment by allo-** [comment]
Okay, the second point is balancing the privacy of user A against the information user B has about his readers, so let's stick to the first point: 

- Allow to mark a user you're following to appreach only in one or more lists, but not in the main timeline.

Especially when there are features against private accounts in lists and other restrictions, it is unfortunate to create a "main" list instead of using your timeline.

Do messages of muted users show up in lists?

**Comment by yogthos** [comment]
I had the same suggestion [here](https://github.com/tootsuite/mastodon/issues/6982). My view is that as long as the person being added to a list is aware of it, and are able to block the person adding them that would address concerns regarding privacy.

My use case for lists is that I have a list for news that consists of bots, and I don't want those toots showing up in my home timeline. Previously, I had a workaround where I could filter toots from the timeline, however the new filtering behavior breaks that. I opened an issue regarding that [here](https://github.com/tootsuite/mastodon/issues/7953).

**Comment by trwnh** [comment]
Yeah, this is a duplicate of #6982, isn't it?

**Comment by naught101** [comment]
I'd also be happy to have a solution where I *can* follow someone, but hide them from my timeline. For example, there are a few news bots that have a very high output of information that I'm sometimes interested in, but I don't want them on my home timeline, because they drown out everything else. Having a dedicated "news" list just for those would solve that problem.

**Comment by Gaffen** [comment]
Interesting fact: If you add accounts to a list, then unfollow the members, they continue to appear in your list...

It also appears to be the case that your ability to edit the list then stops working :/

**Comment by allo-** [comment]
bump

**Comment by zecamigo** [comment]
I would love to see this implemented. Lists would be much more useful in this way.
An easy way to filter out bots I follow and still interact with real people.

**Comment by TatharNuar** [comment]
I saw this issue because I have the same problem as @yogthos brought up.  I have an account I want to follow, but his toots are so much more frequent than all other accounts I'm following combined that I need his toots to show up in a separate list feed instead of Home.  I was hoping that a solution would already exist.

**Comment by RetroYogi** [comment]
Would love to see that feature too. There are many ways in which it could be implemented:
- a "mute from Home timeline" feature (additionally to the current mute feature)
- muted toots simply continue to appear in lists
- an "add or remove from lists" option on the profiles pages of _not followed_ users

**Comment by raphaeljolivet** [comment]
I was expecting this to be the default behavior. 
I tried to create lists to tidy my main timeline. I don't the the point of lists without this feature.

At list it could be an optional a attribute for each list "hide in main timeline"

**Comment by burndive** [comment]
I too would like this to be possible.  It is *essential* to managing the signal to noise ratio.

For now I'm doing to work-around of creating a "main" list with only the accounts I want on my timeline (and I was able to add myself to this list, though I couldn't do that from my profile).

This really only works in the one app I found with proper lists support (Fedilab).  On the web interface, lists are still an afterthought.

**Comment by TimothyStiles** [comment]
Want to bump this as well. Have a bunch of news bots that I want in a separate feed. Trying to play around with what's currently available via mastodon web ui. Does anyone know of a bot that posts very frequently that I could use as a test case?

**Comment by yogthos** [comment]
HN mirror bot is a good example @hackernews@die-partei.social whenever it posts, your timeline ends up being flooded

**Comment by loxK** [comment]
> I don't like the idea of not knowing if someone is tracking me. Sure, they can "track" me in the Local and Federated Timelines, but that's a morass to filter through. You would also likely need to notify an account they have been added to a list, and provide the account a way to prevent that from happening, whether per list or altogether. You'd also need to put in rules for not allowing a blocked account from adding your account to a list, prevent private accounts from being added to any lists or only being added to lists by people they already allowed to follow them, etc. I think a lot of edge cases can be fixed by only allowing followed accounts to be added to lists.

@nightpool I'll be happy with a setting for lists "Do not show in main timeline", that simple. 
Or a filter: "Hide of timeline if added to lists : ..."

**Comment by yogthos** [comment]
@loxK I still don't see why adding somebody to a list can't be treated exactly the same way as a follow from the perspective of the person being added.

**Comment by zecamigo** [comment]
I think the filter function could be further developed with options to include/exclude accounts/accounts in lists.

This would be possible way to sort out the lists/home timeline mess.

**Comment by brendanjones** [comment]
The issue title doesn't quite match the request

The request in the task description to put people in lists without following is 100% a duplicate of https://github.com/mastodon/mastodon/issues/6982. If that's the solution you're set on @allo-  then this task should be closed in favour of #6942 . Though as @nightpool [mentioned](https://github.com/mastodon/mastodon/issues/7876#issuecomment-399695891), putting people in lists without following is probably never going to happen, so #6982 should probably have a `wontfix` on it.

"Add accounts to lists without having them in your main timeline" is achievable in other ways than putting in lists without following, however, so if you're solution-agnostic then change the task description and leave this open.

**Comment by allo-** [comment]
It's just a question of terms. You think of "following" as sending a request to subscribe to their posts, but the usual UIs distinguish between following (see their posts in your timeline) and listing (See the posts in a list) and don't talk about the remote end seeing a subscription request. This issue is mainly about the UX for the subscriber.

**Comment by RetroYogi** [comment]
I don't think that it shall be possible to follow users without them getting a notification about it. This feels kinda like spying on someone.  Also, it is already possible to do something similar by adding user's profile page URLs in an RSS reader. 

But I totally want to clean up my home feed too. There should be a possibility to hide specific user's posts from the home feed and still have them show up in lists.

I currently see two requests in this thread:
1) unclutter the home feed by showing specific users's posts only in lists
2) additional "privacy" / follow someone without them getting notified

Maybe the request would get a better chance of being taken care of if it was more specific? For example focusing only on 1)

**Comment by brendanjones** [comment]
lol awesome, the two issues are now literally having the [same discussion](https://github.com/mastodon/mastodon/issues/6982#issuecomment-1328657912) in parallel. Honestly this one should just be closed, as it was created later and has less comments and reactions. Join the discussion over there on #6982, people.

**Comment by zecamigo** [comment]
#18955 Will probably solve this.

**Comment by burndive** [comment]
Yeah, I'm all for expanding filters, but I don't think expanding filters is a good way of implementing this, and I also think it would be a conceptual and UX nightmarish way to ostensibly provide this functionality.  

It should be dead simple when following someone to add them to one or more lists rather than or in addition to the main timeline.  It should also be simple to manage which lists a followed user is displayed in from their profile.  Mastodon should provide Exclusive Lists as a feature, rather than something that is technically possible with careful planning and knowledge of regular expressions hidden away in a side menu unrelated to the account you're trying to subscribe to.

**Comment by allo-** [comment]
Isn't this most straightforward UI?

Click `Add/Remove from lists`

```
[x] Main timeline
[_] list1
[_] list2
[_] list3
```

Twitter does it another way, but that doesn't have to mean that it's the best way. Treating the main timeline as a "default list" is an easy to understand pattern and also works good with the multi-column view in which the main timeline also is only "yet another column".

**Comment by loxK** [comment]
> > #18955 Will probably solve this.
> 
> Could you (or anyone, I guess) give tl;dr on how? My eyes sort of glazed over trying to understand that link, whereas something like [#16556](https://github.com/mastodon/mastodon/issues/16556) (Exclusive Lists) seems pretty straightforward for this purpose and basically already exists in a fork.

Simple, that would allow using filters to filter out from the timeline, the toots that appear in lists.

**Comment by sunarch** [comment]
I would also like if something like this was implemented.

**Comment by brendanjones** [comment]
As noted here https://github.com/mastodon/mastodon/issues/6982#issuecomment-1412037185, the UI change to enable this is simple, and the code changes required are likely solved somewhat by https://github.com/mastodon/mastodon/pull/22048.

**Comment by trwnh** [comment]
Duplicate of #6982 or #16556 or possibly something that could be done with filters that apply to home but *not* lists (#18955)
